packages/builder/src/components/nav/ModelNavigator/ModelNavigator.svelte

```diff
@@ -35,6 +35,10 @@
   }
 
   function setupForNewModel() {
+    setTimeout(() => {
```

Comment:
Same for this one. If you are relying on a state update to complete and elements to be available on the screen before you execute your next step - you may need to use a `tick`.

https://svelte.dev/tutorial/tick